### ISSUE TITLE
运行结果中test_results这个文件夹里的pdf文件0 20 40 60 80 都是些什么，为什么在result中的结果很差，但是这个test_results中的图片看起来就很好呢

### ISSUE THREAD

**Issue by kooper9934** [MAZiqing/FEDformer#78]

**Comment by FAFUuser** [comment]
0 20 40 60 80 是测试集上的结果，作者在代码中将测试集上每20个迭代次数的真实值与预测值绘制出来，并保存为pdf。
  def test(self, setting, test=0):
        ##
        ....... 
        ##
        with torch.no_grad():
            for i, (batch_x, batch_y, batch_x_mark, batch_y_mark) in enumerate(test_loader):
                pred, true = self._process_one_batch(test_data, batch_x, batch_y, batch_x_mark, batch_y_mark)
                preds.append(pred.detach().cpu().numpy())
                trues.append(true.detach().cpu().numpy())
                if i % 20 == 0:
                    input = batch_x.detach().cpu().numpy()
                    gt = np.concatenate((input[0, :, -1], true[0, :, -1]), axis=0)
                    pd = np.concatenate((input[0, :, -1], pred[0, :, -1]), axis=0)
                    visual(gt, pd, os.path.join(folder_path, str(i) + '.pdf'))

**Comment by tianzhou2011** [comment]
和上面同学说的一样，就是每20个batch存下来的sample
